specs/Menu.md

```diff
@@ -1,731 +1,545 @@
+| Mouse    | Click      | Dismiss | Click anywhere outside the component                                   |
+| Mouse    | Click      | Dismiss | Click on the trigger while the menu is open                            |
+| Mouse    | Click      | Dismiss | Click on a menu item                                                   |
+| Mouse    | MouseLeave | Dismiss | Mouse leaves the component after a delay                               |
+| Keyboard | Enter      | Dismiss | Invoked on a menu item                                                 |
+| Keyboard | Space      | Dismiss | Invoked on a menu item                                                 |
+| Keyboard | Esc        | Dismiss | Closes the menu and focuses on the triggering element                  |
+| Keyboard | Tab        | Dismiss | Closes the menu and all submenus, focus moves to next element in order |
 
-##### Focus
+### Submenu trigger/navigation
 
-The menu item should change border and background color on focus.
+A submenu can be triggered by the following user interactions on the triggering menu item. Not all interactions should be supported at the same time, but the component must be able to support combinations of the below interactions.
 
-##### Hover
+As a general rule, once a submenu is dismissed without dismissing the menu, the focus should revert to the triggering menu item unless the interaction involves another focusable UI component.
 
-The menu item should change border and background color on hover.
+| Type     | Action     | Result  | Details                                                                |
+| -------- | ---------- | ------- | ---------------------------------------------------------------------- |
+| Mouse    | Click      | Open    | Click the menu item that contains a submenu                            |
+| Mouse    | Hover      | Open    | Hover over the menu item that contains a submenu with delay            |
+| Keyboard | Enter      | Open    | Focus on triggering menu item                                          |
+| Keyboard | Space      | Open    | Focus on triggering menu item                                          |
+| Keyboard | ArrowRight | Open    | Focus on triggering menu item                                          |
+| Mouse    | Click      | Dismiss | Click on an item in the submenu                                        |
+| Mouse    | Click      | Dismiss | Click on a UI element that is not the submenu                          |
+| Mouse    | MouseLeave | Dismiss | Mouse leaves the submenu or its triggering menu item after delay       |
+| Keyboard | ArrowLeft  | Dismiss | Closes the submenu and focuses on the triggering menu item             |
+| Keyboard | Esc        | Dismiss | Closes the submenu and focuses on the triggering menu item             |
+| Keyboard | Tab        | Dismiss | Closes the menu and all submenus, focus moves to next element in order |
 
```

Comment:
also see https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html# for extensive description of left/right arrow keys for submenu interaction

Reply:
That is exactly where most of this comes from, I will add a reference link to the section

Reply:
Added a references section to this chapter